### PR TITLE
feat: change welcome email's copy

### PR DESCRIPTION
**Pull request by migcarva**
This PR introduces the following changes:

* new copy
* renames the test file to represent the test naming convention of the project
* tests check for CTA to Activate Account and mail to Support

### CODE REVIEW

File: packages/react-components/src/messages/__tests__/Welcome.test.tsx [hunk 1]
```diff
@@ -14,5 +14,15 @@ it('renders the welcome template with name and link', () => {
...
   const heading = getByRole('heading');
+  const cta = getByRole('link', {
+    name: /activate account/i,
+  }) as HTMLAnchorElement;
+  const mailToSupport = getByRole('link', {
+    name: /get in touch/i,
+  }) as HTMLAnchorElement;
+
   expect(heading.textContent).toContain('John Doe');
+  expect(cta.href).toBe('https://example.com/');
```
Comment (jeysal) [inline packages/react-components/src/messages/__tests__/Welcome.test.tsx:25]: split into multiple test cases with descriptive titles please — one concern per test case

File: packages/react-components/src/messages/__tests__/Welcome.test.tsx [hunk 2]
```diff
@@ -14,5 +14,15 @@ it('renders the welcome template with name and link', () => {
   );
 
   const heading = getByRole('heading');
+  const cta = getByRole('link', {
+    name: /activate account/i,
+  }) as HTMLAnchorElement;
```
Comment (jeysal) [inline packages/react-components/src/messages/__tests__/Welcome.test.tsx:19]: To make this case safe, the `selector: 'a'` option to getByRole should also be used

**Review by jeysal — CHANGES_REQUESTED** [review]
couple of test comments other than that lgtm

File: packages/react-components/src/messages/__tests__/Welcome.test.tsx (hunk 1 above)
Comment (migcarva) [reply packages/react-components/src/messages/__tests__/Welcome.test.tsx]: Multiple test cases aded

File: packages/react-components/src/messages/__tests__/Welcome.test.tsx (hunk 2 above)
Comment (migcarva) [reply packages/react-components/src/messages/__tests__/Welcome.test.tsx]: Can you check the updated version?

**Review by jeysal — APPROVED** [review]

**Pull request merged**